src/components/code/index.tsx

```diff
@@ -61,7 +71,7 @@ const Code: FC<CodeProps> = ({
   };
 
   return (
-    <Flex width="100%" sx={{ ...styles }} height="100%">
+    <Flex width="100%" sx={{ ...styles, p: padding }} height="100%">
```

Comment:
does it make sense to rather take the `sx` from the props and do a `sx={{...styles, ...sx}}`?
it would enable a native sx thing with `p="20px"`

Reply:
Yes, and we are slowly moving from sx by the way